packages/expo/src/dom/webview-wrapper.tsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { AppState } from 'react-native';
 
 import { getBaseURL } from './base';
-import type { BridgeMessage, DOMProps, WebViewProps, WebViewRef } from './dom.types';
+import type { BridgeMessage, WebViewProps, WebViewRef, DOMProps } from './dom.types';
```

Comment:
```suggestion
import type { BridgeMessage, DOMProps, WebViewProps, WebViewRef } from './dom.types';
```

not sure what's the intention to put DOMProps at last

Reply:
No intention, only left over from adding DOMPropsInternal in previous revision.